cmd/subnetcmd/deploy.go

```diff
@@ -104,26 +110,161 @@ func deploySubnet(cmd *cobra.Command, args []string) error {
 		network = models.NetworkFromString(networkStr)
 	}
 
+	// deploy based on chosen network
 	ux.Logger.PrintToUser("Deploying %s to %s", chains, network.String())
-	// TODO
+	chain := chains[0]
+	chainGenesis := filepath.Join(app.GetBaseDir(), fmt.Sprintf("%s_genesis.json", chain))
+
 	switch network {
 	case models.Local:
 		app.Log.Debug("Deploy local")
-		// TODO: Add signal management here. If we Ctrl-C this guy it can leave
-		// the gRPC server is a weird state. Should kill that too
-		deployer := subnet.NewLocalDeployer(app)
-		chain := chains[0]
-		chainGenesis := filepath.Join(app.GetBaseDir(), fmt.Sprintf("%s_genesis.json", chain))
-		err := deployer.DeployToLocalNetwork(chain, chainGenesis)
-		if err != nil {
+		deployer := subnet.NewLocalSubnetDeployer(app)
```

Comment:
Because it refers to an old version, unless I am confusing something. If we do Ctrl-C now it **will** kill the server _unless_ started from a different run (`BackendStartedHere`).